README.md

```diff
@@ -1,2 +1,256 @@
-# Hello-world
-This Repo is for code review
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+-   [📖 About the Project](#about-project)
+    -   [🛠 Built With](#built-with)
+        -   [Tech Stack](#tech-stack)
+        -   [Key Features](#key-features)
+    -   [🚀 Live Demo](#live-demo)
+-   [💻 Getting Started](#getting-started)
+    -   [Setup](#setup)
+    -   [Prerequisites](#prerequisites)
+    -   [Install](#install)
+    -   [Usage](#usage)
+    -   [Run tests](#run-tests)
+    -   [Deployment](#deployment)
+-   [👥 Authors](#authors)
+-   [🔭 Future Features](#future-features)
+-   [🤝 Contributing](#contributing)
+-   [⭐️ Show your support](#support)
+-   [🙏 Acknowledgements](#acknowledgements)
+-   [❓ FAQ (OPTIONAL)](#faq)
+-   [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hellow_World] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hellow_World]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
```

Comment:
- [ ]  Kindly make sure you only list the stack you have used to build up your project which are HTML, and CSS in this case. and delete that that does not apply to make your README looks more professional